src/Post.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import User from './User';
+import Comment from './Comment'
+
+class Post extends React.Component {
+    constructor(props) {
+        super(props)
+        this.state = {
+            isOpenComments: false
+        }
+    }
+
+    openComments = () => {
+        return this.state.isOpenComments ?
+            this.setState({ isOpenComments: false }) :
+            this.setState({ isOpenComments: true })
```

Comment:
`this.setState((prevState) => ({ isOpenComments: !prevState.isOpenComments }));`